apps/cowswap-frontend/src/modules/swap/services/ethFlow/index.ts

```diff
@@ -34,6 +37,12 @@ export async function ethFlow(
     return undefined
   }
 
+  if (appDataContainsHooks(orderParamsOriginal.appData.fullAppData)) {
+    reportAppDataWithHooks(orderParamsOriginal)
```

Comment:
i guess this should never happen, maybe add a comment so its removed later when we find the real issue. It seems we do the checks multiple times (at least 3 places check for ethFlow and remove the hooks, makes it too hard to reason about).

Reply:
972db4f606ebabe64840e84257e9b8f8ca3e3c3c